plugins/inputs/cadvisor/mesos/plugin.go

```diff
@@ -0,0 +1,41 @@
+// Copyright 2019 Google Inc. All Rights Reserved.
```

Comment:
I'm just going to have to trust this. This is all lifted from another plugin, right?

Reply:
Same argument with me.

Reply:
I just copied this directly from cadvisor.

Reply:
There are a lot of instances where we directly take plugin source code and drop it into our repo. Is that a requirement? Is there no way to reference it externally and import it?